spec/services/weather_service_spec.rb

```diff
@@ -50,10 +50,10 @@
     lat = 39.738453
     lon = -104.984853
     body = service.get_weather(lat, lon)
-
+    
     5.times do |index|
       expect(body[:daily][index][:dt].is_a?(Numeric)).to be true
-      expect(body[:daily][index][:rain].is_a?(Numeric)).to be true
+      expect(body[:daily][index][:rain].is_a?(Numeric)).to be true if body[:daily][index][:rain]
```

Comment:
if it isnt going to rain, there wont be a rain value. running `to_i` so `nil.to_i` = 0 which is exactly what I want